README.md

```diff
@@ -10,43 +10,30 @@ Clone the repo:
 git clone https://github.com/NREL/mappymatch.git
 ```
 
-Then, setup a python environment with python >= 3.8:
+Change directory into the mappymatch folder:
 ```
-conda create -n mappymatch python=3.8
+cd path/to/mappymatch   # from here, your path to mappymatch is likely .\mappymatch\
 ```
 
-Finally, use pip to install the package:
+Then, use the environment.yml file (which was downloaded when you cloned the repo) to install dependencies:
+```
+conda env create -f environment.yml
+```
+
+To activate the mappymatch environment:
 ```
 conda activate mappymatch
-pip install -e <path/to/mappymatch> 
 ```
 
 There are a couple of extras available:
 
-* `osrm`: Needed by the `OsrmMatcher` to connect to the OSRM API.
 * `plot`: Needed to plot the result.
 
 At least `plot` is needed to run the examples.
 
-### Alternate Methods
-
-If you have issues installing the package and dependencies using pip you can try using conda to
-install the dependencies:
-
-Clone the repo:
-```
-git clone https://github.com/NREL/mappymatch.git
-```
-
-Then, use the provided environment.yml file to install dependencies:
-```
-conda env create -f environment.yml
-```
-
-Finally, use pip to install the package:
+This can be installed via pip:
 ```
-conda activate mappymatch
-pip install -e <path/to/mappymatch> 
+pip install plot
```

Comment:
```suggestion
pip install ".[plot]"
```

Just tested the above on my machine and it seems to be working. [Link to syntax example](https://stackoverflow.com/a/43090648)